doc/internationalization.md

```diff
@@ -0,0 +1,44 @@
+gRPC Internationalization
+=========================
+
+As a universal RPC framework, gRPC needs to be fully usable within/across different international environments. 
+This document describes gRPC API and behavior specifics when used in a non-english environment.
+
+## API Concepts
+
+While some API elements need to be able to represent non-english content, some are intentionally left as ASCII-only
+for simplicity & performance reasons.
+
+### Method name (in RPC Invocation)
+Method names are ASCII-only. Most gRPC services will use protobuf which only allows ASCII based method names anyway.
```

Comment:
Should we mention that the ASCII strings need to also be HTTP/2 header compliant?